portal/src/main/webapp/js/src/crosscancer.js

```diff
@@ -155,7 +192,17 @@
                                     studies: histDataOrg
                                 }
                             })).render();
-                            var histData = filterAndSortData(histDataOrg);
+                            
+                            
+                            var cancerTypes = _.uniq(_.map(histDataOrg, function(e){return metaData.type_of_cancers[metaData.cancer_studies[e.studyId].type_of_cancer];}));
+                            $.each(cancerTypes, function(key, value) {   
```

Comment:
I would suggest to use _.each jQuery each function always has performance issue.